test/unit/toUint8Array.js

```diff
@@ -0,0 +1,103 @@
+'use strict';
+
+const assert = require('assert');
+
+const sharp = require('../../');
+const fixtures = require('../fixtures');
+
+describe('toUint8Array', () => {
+  beforeEach(function () {
+    sharp.cache(false);
+  });
+  afterEach(function () {
+    sharp.cache(true);
+  });
+
+  it('should output data as Uint8Array', (done) => {
+    const image = sharp(fixtures.inputJpg320x240);
+    image.toUint8Array().then((data) => {
+      assert.strictEqual(data instanceof Uint8Array, true);
+      assert.strictEqual(data.length, 26641);
+      done();
+    });
```

Comment:
Tests using the callback style (via `done`) that also use Promise chains will need a `.catch(done)` on the chain to ensure the test runner is made aware of any errors.

An alternative is to remove the `done` callback entirely and return the Promise chain.